INSTALL.txt

```diff
@@ -0,0 +1,39 @@
+# INSTALL / BUILD instructions for Apache Superset (incubating)
+
+# These instructions should work along sources releases under
+# https://dist.apache.org/repos/dist/dev/incubator/superset/*/*source.tar.gz$
+
+# [required] fetch the tarball and untar the source
+# change into the directory that was untarred.
+
+#############################
+# Dependencies
+#############################
+# Superset runs best on python3.6
+# Also recommended is the latest long term supported node / npm
+# For complementary information read our CONTRIBUTING.md
+# for OS level dependencies find instructions here:
+# http://superset.apache.org/installation.html
+
+# Fetch npm dependencies
+cd superset/assets/
+npm ci
+
+# Build the Javascript bundles
+npm run prod
```

Comment:
Do we use this or `npm run build`?

Reply:
oh right, `npm run build` is best I think, altering